Tests/LinuxMain.swift

```diff
@@ -4,4 +4,5 @@ import FormworksTests
 
 var tests = [XCTestCaseEntry]()
 tests += FormworksTests.allTests()
+tests += AutoLayoutTests.allTests()
```

Comment:
You missed one renaming here.

Reply:
I've pushed the changes. This mistake didn't result in a failed CI since it is running on macOS. We should be careful to always keep this file update to date.